Error running Translations action

Error when running translations:

```
Run qgis-plugin-ci push-translation 
usage: qgis-plugin-ci push-translation [-h] transifex_token
qgis-plugin-ci push-translation: error: the following arguments are required: transifex_token
Error: Process completed with exit code 2.
```

--- a/.github/workflows/tests.yml
+++ b/.github/workflows/tests.yml
@@ -25,7 +25,7 @@ jobs:
           submodules: true
 
       - name: Pull qgis image
-        run: docker pull qgis/qgis:final-3_14_1_focal
+        run: docker pull qgis/qgis:final-3_14_15_focal
 
       - name: Pull PostGIS image
         run: docker pull kartoza/postgis:12.0
@@ -35,4 +35,4 @@ jobs:
 
       # Runs all tests
       - name: Run tests
-        run: docker run --rm --net=host --volume `pwd`/Qaava:/app -w=/app -e QGIS_PLUGIN_IN_CI=1 qgis/qgis:final-3_14_1_focal sh -c "pip3 install -q pytest && xvfb-run -s '+extension GLX -screen 0 1024x768x24' pytest -v"
+        run: docker run --rm --net=host --volume `pwd`/Qaava:/app -w=/app -e QGIS_PLUGIN_IN_CI=1 qgis/qgis:final-3_14_15_focal sh -c "pip3 install -q pytest && xvfb-run -s '+extension GLX -screen 0 1024x768x24' pytest -v"

